tools/osbuilder/rootfs-builder/fedora/config.sh

```diff
@@ -5,7 +5,7 @@
 
 OS_NAME="Fedora"
 
-OS_VERSION=${OS_VERSION:-30}
+OS_VERSION=${OS_VERSION:-34}
```

Comment:
Is this needed to get the umoci and skopeo packages?

Reply:
I think Dave said that The version of skopeo in 30 is very back level and Fedora 30 is end of life: #2116